### DISCUSSION TITLE
Question about Name Change

### DISCUSSION THREAD

**Discussion by codiebooboo** [julelang/jule#21] [Discussions]
Why did the Name change from "X" to "Jule"?

**Comment by mertcandav** [comment]
Hi,

The "X" was chosen for letter naming, like most languages. The name "X" had a number of problems, it wasn't unique enough. This was clearly felt in getting domain or in matters such as GitHub organization naming. Due to such reasons, it was decided to change the name of the language in order to prevent some possible obstacles in the future.
